### DISCUSSION TITLE
obsidian-github-publisher vs digital-graden

### DISCUSSION THREAD

**Discussion by makerjackie** [ObsidianPublisher/obsidian-github-publisher#124] [General]
I found another obsidian publish plugin: [digitalgraden](https://dg-docs.ole.dev/) similar to obsidian-github-publisher. 

just for compare, I publish both my blog in these two plugin, This is just a friendly comparison, there is no malice.

- https://obsidian-github-publish.vercel.app
- https://digitalgarden-lime.vercel.app/

I think both have their advantages, but I would slightly lean towards Digital Garden because there are more configurable options and richer themes. However, I still default to using obsidian-github-publisher because I am more familiar with the style of mkdocs.

**Comment by Mara-Li** [comment]
Good to you then

**Comment by Mara-Li** [comment]
Just to be clear : I created Publisher based on some code of digital garden. But when Digital Garden has a preconfigured template FOR the plugin, and the two **must be used** altogether, I have chosen to create a plugin for more general purpose, allowing you to create **your** template, but also, simply send file on a GitHub repository. 

I, for example, use my own plugin to send some file on the wiki of a Roleplay, converting a dataview table for the index page. 

My Plugin is **general**. You can create your own template, and blog. You aren't block with mkdocs. I just chose mkdocs because of I code in python and can create a maximum support of some obsidian things.
Digital Garden is for creating *one* blog, as Publish does.
